modules/openapi-generator/src/main/resources/swift4/CodableHelper.mustache

```diff
@@ -10,62 +10,66 @@ import Foundation
 public typealias EncodeResult = (data: Data?, error: Error?)
 
 open class CodableHelper {
+    
+    private static var customDateFormatter: DateFormatter?
+    private static var defaultDateFormatter: DateFormatter = {
```

Comment:
Instead of having `customDateFormatter` and `defaultDateFormatter` what do you think about having only a `public static var dateFormatter: DateFormatter = {}()` and if anyone wants to use a different one, just changes this , since it's already a variable?

Reply:
The same applies to `customJSONDecoder`, `defaultJSONDecoder` and `customJSONEncoder`, `defaultJSONEncoder`.

Reply:
The reason for having both a defaultDateFormatter and customDateFormatter, is if someone wanted to, just briefly, change from the defaultDateFormatter to a custom one, and then back again (because one endpoint has a custom date format), then that would be possible.